src/pages/auth/login.tsx

```diff
@@ -29,22 +30,39 @@ const LoginPage = () => {
     setEmail(e.target.value);
   };
 
-  const handleSubmit = (e: React.ChangeEvent<HTMLInputElement>) => {
+  const handleSubmit = async (e: React.ChangeEvent<HTMLInputElement>) => {
     e.preventDefault();
+    setIsLoading(true);
 
-    if (email === testAcount.email) {
-      setEmailInvalid(false);
-    } else {
-      setEmailInvalid(true);
-    }
-    if (password === testAcount.password || password.length <= 5) {
-      setPasswordInvalid(false);
-    } else {
-      setPasswordInvalid(true);
-    }
-
-    if (email === testAcount.email && password === testAcount.password) {
-      alert(`Login maded with sucess. Welcome ${email}!!`);
+    try {
+      const response = await UserService.login({ email, password });
+      setIsLoading(false);
+      if (emailValidation(email) === false) {
+        setEmailInvalid(true);
+      }
+      if (passwordValidation(password) === false) {
+        setPasswordInvalid(true);
+        handleToast(
+          'Password must have min 5 and max 20 characters with 1 letter, 1 number and 1 special character',
+          'warning'
+        );
+      }
+      if (!response.user) {
+        setEmailInvalid(true);
+        setPasswordInvalid(true);
+      } else if (response.user.email === email) {
+        setEmailInvalid(false);
+        setPasswordInvalid(false);
```

Comment:
Do we need it? The validations are removed when the user types something in the fields, right? 
If they're not valid, the API should not even be called